appengine/flexible/api_gateway/gateway/api_gateway.py

```diff
@@ -0,0 +1,46 @@
+import requests
+import sys
+from flask import Flask
+app = Flask(__name__)
+
+#development ports {static: 8003, flask1: 8001, flask2: 8002}
+@app.route('/test')
+def test():
+    return "GATEWAY OPERATIONAL"
+
+@app.route('/')
+def root():
+    res = requests.get('https://static-dot-flask-algo.appspot.com/')
+    return res.content
+
+@app.route('/hello/<service>')
+def say_hello(service):
+    services = {
+        'flask1': { 'url': 'https://flask1-dot-flask-algo.appspot.com/', 'send': False },
```

Comment:
@jonparrott  The URLs will depend on the project ID of when the user deploys the demo, so he will have to fill those in. (Unless there is another way to handle that?)  I could just leave a comment there as a placeholder and add something in the readme to instruct the user to enter his urls there.  Would that be good?